script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -276,22 +276,22 @@ The woman was leaving.
 You should go back to your room and do exactly as the doctor says."
 <Page76>
 "Ah―――"
-The woman walked away as if it was a matter of course.
+The woman walked away as if it were a matter of course.
 <Page77>
 "...See you tomorrow."
-Tomorrow we can talk again, just like today.
-I was glad. For the first time since I woke up from the accident, I felt human again.
+Tomorrow we can talk again, just like today. //Tense, self talking, single quite?
```

Comment:
> また明日、今日みたいな話ができる。

Reply:
yeah, I say this just stays as-is. It's clear without single quotes imo